### DISCUSSION TITLE
When the release will be published?

### DISCUSSION THREAD

**Discussion by rwp0** [StrawberryPerl/Perl-Dist-Strawberry#120] [General]
### Links

The links below show 5.32.1 as the latest available Perl release (since apparently not updated).

- https://strawberryperl.com/
- https://community.chocolatey.org/packages/StrawberryPerl (to be installed with `choco`)
- https://winstall.app/apps/StrawberryPerl.StrawberryPerl (to be installed with `winget`)

Please publish the 5.38 (latest) release through these media above, so users can download and install easily.

And probably 5.36.1 too, since that's also released on github jointly with 5.38:

- https://github.com/StrawberryPerl/Perl-Dist-Strawberry/releases/tag/SP_5380_5361

Thanks

**Comment by wchristian** [comment]
See https://github.com/StrawberryPerl/Perl-Dist-Strawberry/issues/114#issuecomment-1632064067
